draft-ietf-quic-applicability.md

```diff
@@ -416,6 +416,11 @@ connection. While the long header always exposes some information (such as the
 version and Connection IDs), the short header exposes at most only a single
 Connection ID.
 
+Note that the Connection ID in the short header may be omitted, requested via
+the omit_connection_id transport parameter. However, even if requested, the
```

Comment:
How old is this PR?  omit_connection_id was removed in draft-ietf-quic-transport-09.

Reply:
Ups, this was meant to address issue #23 which was like to an old PR... we will fix that.